hr_timesheet_purchase_order/models/hr_timesheet_recurrence.py

```diff
@@ -0,0 +1,399 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from calendar import monthrange
+
+from dateutil.relativedelta import relativedelta
+from dateutil.rrule import (
+    DAILY,
+    FR,
+    MO,
+    MONTHLY,
+    SA,
+    SU,
+    TH,
+    TU,
+    WE,
+    WEEKLY,
+    YEARLY,
+    rrule,
+)
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+MONTHS = {
```

Comment:
This hardcoded part should be used from corresponding library, `calendar.monthrange`

Reply:
Thanks, for feedback, I'll fix it